metaflow/plugins/argo/argo_workflows_deployer.py

```diff
@@ -257,6 +259,7 @@ class ArgoWorkflowsDeployer(DeployerImpl):
     """
 
     TYPE: ClassVar[Optional[str]] = "argo-workflows"
+    TIMEOUT: ClassVar[Optional[int]] = 10
```

Comment:
this may not be needed anymore given #1980 is already merged and released

Reply:
Rebased PR to pull in latest changes.